MoreSeamothUpgrades/MoreSeamothUpgrades.csproj

```diff
@@ -50,17 +50,19 @@
     <Reference Include="System.Data" />
     <Reference Include="System.Xml" />
     <Reference Include="UnityEngine">
-      <HintPath>D:\Steam Library\steamapps\common\Subnautica\Subnautica_Data\Managed\UnityEngine.dll</HintPath>
+      <HintPath>D:\Steam\steamapps\common\Subnautica\Subnautica_Data\Managed\UnityEngine.dll</HintPath>
```

Comment:
As a courtesy to the author, revert changes to DLL reference paths in `csproj` files before submitting the Pull Request.